src/components/Navigation/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import Container from 'src/components/Container';
+import Logo from 'src/components/Logo';
+import MenuMain from 'src/components/MenuMain';
+
+import {
+  FixedWrapper,
+  InnerBox,
+  LogoBox,
+  MenuBox,
+  DesktopRightBox,
+} from './style';
+
+export interface IProps {}
```

Comment:
Зачем  пустой интерфейс ?
```suggestion
export interface IProps {}
```

Reply:
оставил на момент работы над анимации навигационной панели. но ты прав - пока удалю